### PR TITLE
New colormap

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

New colormap to easily display distance maps.

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [ ] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: src/DGtal/io/colormaps/SimpleDistanceColorMap.h [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+ **/
+
+#pragma once
+
+/**
+ * @file SimpleDistanceColorMap
+ * @author David Coeurjolly (\c david.coeurjolly@liris.cnrs.fr )
+ * LIRIS (CNRS, UMR 5205), Université de Lyon, France
+ *
+ * @date 2011/07/18
```
Comment (kerautret) [inline src/DGtal/io/colormaps/SimpleDistanceColorMap.h]: ;)

File: src/DGtal/io/colormaps/SimpleDistanceColorMap.h [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+     * @return a reference on 'this'.
+     * Forbidden by default.
+     */
+    SimpleDistanceColorMap & operator= ( const SimpleDistanceColorMap & anOther );
+    
+    
+    /**
+     * Constructor from two min and max values.
+     *
+     * @param[in] amin the minimum value of the range.
```
Comment (kerautret) [inline src/DGtal/io/colormaps/SimpleDistanceColorMap.h]: uniform the @param[in] everywhere ?

File: src/DGtal/io/doc/moduleBoard2D.dox [hunk 3]
```diff
@@ -1,5 +1,5 @@
```
Comment (kerautret) [inline src/DGtal/io/doc/moduleBoard2D.dox]: in the image DGtal/io/doc/images/colormaps.png a part of the text on the left looks hidden, if you have the possibility to shift it in the left it should be nice:
colormaps

Comment (dcoeurjo) [reply src/DGtal/io/doc/moduleBoard2D.dox]: Thanks.. fixed

**Comment by dcoeurjo** [comment]
I'm merging this PR. thanks @kerautret for the comments. There are still issues on travis but not related to this one.

**Pull request merged**